Firestore/Protos/CMakeLists.txt

```diff
@@ -102,10 +102,10 @@ cc_library(
 
 target_include_directories(
   firebase_firestore_protos_libprotobuf PUBLIC
-    ${FIREBASE_SOURCE_DIR}/Firestore/Protos/cpp
+  ${FIREBASE_SOURCE_DIR}/Firestore/Protos/cpp
 )
 
 set_target_properties(
   firebase_firestore_protos_libprotobuf PROPERTIES
-    COMPILE_FLAGS "-Wno-unused-parameter"
+  COMPILE_FLAGS "-Wno-unused-parameter"
```

Comment:
Let's address style separately if you're still concerned.

Reply:
> Let's address style separately if you're still concerned.

Nope; I like that style.

> My baseline position is that without automatic formatting support, I'm disinclined to make things complicated.

I'd generally go a bit further and assume that without auto format support, we simply don't care enough for this to matter. (If we did, we'd add it.) *Maybe* less relevant for cmake, since we'd likely have to write it ourselves.